### DISCUSSION TITLE
[Prettier Plugin] Custom theme width gets sorted directly to the beginning of the class list

### DISCUSSION THREAD

**Discussion by bjesuiter** [tailwindlabs/tailwindcss#11459] [Help] [answered]
I tried to add a custom width in tailwind config like this: 

```
 theme: {
    extend: {
      width: {
        90: "22rem",
      },
    },
  },
```

When I try to use this, it gets sorted like this: 

```jsx
<header class="md:h-90 h-80 w-full bg-cover bg-center bg-no-repeat">
</header>
```

The problem is, that now the h-80 takes precedence over the md:h-90, even when the breakpoint reacts. 

Do I do something wrong here or is this broken?

**Comment by wongjn** [comment, marked as answer]
It could be that the Prettier plugin sees `md:h-90` as a custom class due to the fact that `h-90` does not seem to exist as a Tailwind class candidate. You may have meant to use a `width` utility class, like `md:w-90` or you may have meant to extend `height` instead of `width`.

**Reply by bjesuiter** [reply]
Ohh, such a stupid error. 🤦‍♂️ I wanted to extend height, not width. 
Thank you very much! <3
